compiler/code-gen/declarations.cpp

```diff
@@ -1068,10 +1060,6 @@ void ClassDeclaration::compile_job_worker_shared_memory_piece_methods(CodeGenera
 }
 
 void ClassMembersDefinition::compile(CodeGenerator &W) const {
-  if (G->is_output_mode_k2()) {
```

Comment:
same as for `ClassDeclaration::compile_accept_visitor_methods`

Reply:
+